### DISCUSSION TITLE
State of HmIP-RC8 channels is not available

### DISCUSSION THREAD

**Discussion by briemla** [SukramJ/aiohomematic#1350] [Q&A] [answered]
Hi, we use some homematic IP components in our house. One of the components is the `HmIP-RC8`. Two of its channels are connected to other homematic IP components to turn on and off lights. Now we want to use the remaining channels via home-assistant  and the custon_homematic extension to trigger some other stuff. Unfortunately, the state of the buttons is unknown. There are also no entries in the log after one of the buttons is triggered. Can anybody help with this issue?

We use a CCU3, version 3.71.12 and home-assistant-core version 2023.12.4 and custom_homematic version 1.52.0

**Comment by SukramJ** [comment, marked as answer]
Please read the corresponding [chapter](https://github.com/danielperna84/custom_homematic#events-for-homematicip-devices) in the docs. HmIP devices must be activated in the CCU to send events.

State unknown is also ok untill an events has been received, afterwards it's the timestamp of the last activation. More information can be found [here](https://developers.home-assistant.io/docs/core/entity/event).

**Reply by briemla** [reply]
Thanks a lot that helped. Sorry, for not reading the documentation about it.
